### ISSUE TITLE
how to mount WD Mybook using generic SATA to USB enclosure

### ISSUE THREAD

**Issue by azrielnation** [themaddoctor/linux-mybook-tools#18]
I have WD Mybook 4 Tb, while the drive still perfectly working, the board give up suddenly. I still can access data from windows just like about 5-10 sec before the board disconnect the connection. I have try to dissemble the drive and bought a generic SATA to USB enclosure. But the disk said not formatted and cannot opened. I never set any password on it.

I'm not familiar with Linux, but trying follow the instruction one by one, but get lost because I got different result from first step.

Is the data really encrypted by onboard chip WD even I don't use their tools to encrypt? or I just need to repair the MBR just to access my file? thanks for your help.

I already hopeless and want to buy another WD Mybook just to get the working board. But unfortunately, my HDD type already discontinue. The board code is 4061-705149-A00 Rev AB. If you know that the newest type can be use as substitute, I will take that option. But if the encryption really specific by board code, then no option left for me unless you help me step by step to retrieve/ mount the drive. thanks again.

**Comment by themaddoctor** [comment]
Yes, it is encrypted. Setting a password only changes how the key is stored.
